pycbc/workflow/core.py

```diff
@@ -1517,8 +1518,7 @@ def to_segment_xml(self, override_file_if_exists=False):
         # create XML doc and add process table
         outdoc = ligolw.Document()
         outdoc.appendChild(ligolw.LIGO_LW())
-        process = ligolw_utils.process.register_to_xmldoc(outdoc, sys.argv[0],
-                                                          {})
+        process = ligolw_process.register_to_xmldoc(outdoc, sys.argv[0], {})
```

Comment:
This basically all looks good to me, but I'm confused by this change. As far as I know, nothing changed here and this code is being used by the all-sky generator, apparently without problem, so why do we need to switch from ligolw_utils to ligolw_process. Are you maybe using the lalsuite glue and not pycbc-glue?